### PR TITLE
Add 4 Digit osu!mania World Cup 2023 original tracks into osu! originals

### PR DESCRIPTION
**Pull request by indekkusu-era**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] Native review

### CODE REVIEW

**Comment by Walavouchey** [comment]
@juankristal @Pisapou quick review maybe

File: wiki/osu!_originals/en.md [hunk 1]
```diff
@@ -336,7 +351,6 @@ These songs were later [announced](https://osu.ppy.sh/home/news/2023-02-01-twin-
 | [blobdash - Resentment](https://soundcloud.com/blobdash/resentment) | [#1](https://osu.ppy.sh/beatmapsets/1850544) | Semifinals FM2 |
 | [blobdash & Raphlesia - Recollection](https://soundcloud.com/blobdash/recollection) | [#1](https://osu.ppy.sh/beatmapsets/1828727) | Round of 96 tiebreaker |
 | [DJ Raisei - Fractured Angel](https://www.youtube.com/watch?v=YlaYNIXJAyM)[^fa] | [#1](https://osu.ppy.sh/beatmapsets/1838114) | Round of 32 tiebreaker |
-| [DJ Witch Girl MIR4CLEN - Danrit](https://soundcloud.com/mir4clen/osutaiko) | [#1](https://osu.ppy.sh/beatmapsets/1842746) | Round of 16 FM3 |
```
Comment (juankristal) [inline wiki/osu!_originals/en.md:339]: Why is this being removed? 🤔

File: wiki/osu!_originals/es.md [hunk 2]
```diff
@@ -223,6 +223,21 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
 | :-: | :-: | :-- |
 | [BilliumMoto x Akiri - Cryochemistry](https://soundcloud.com/billiummoto/cryochemistry)[^fa] | [#1](https://osu.ppy.sh/beatmapsets/1698378) | Desempate de la gran final |
 
+#### ![](/wiki/shared/mode/mania.png) [4 Digit osu!mania World Cup 2023](/wiki/Tournaments/4DM/2023)
+
+| Canción | Beatmap | Notas |
+| :-: | :-: | :-- |
+| [Akiri vs. HowToPlayLN - Cognitohazadrous Ocular Vision Impairing Disease](https://soundcloud.com/indekkusu/akiri-vs-howtoplayln-cognitohazadrous-ocular-vision-impairing-disease) |  | Cutscene background music |
+| [HowToPlayLN - Lost in Space](https://soundcloud.com/indekkusu/4dm2023-trailer) |  | Trailer music |
+| [HowToPlayLN - monochroma.](https://soundcloud.com/indekkusu/monochroma) |  | Cutscene background music |
```
Comment (juankristal) [inline wiki/osu!_originals/es.md:232]: Translation for this should be "Canción de intervalo" I believe.

File: wiki/osu!_originals/es.md [hunk 3]
```diff
@@ -223,6 +223,21 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
...
+| [HowToPlayLN - wandering through the stars](https://soundcloud.com/indekkusu/wandering-through-the-stars) |  | Cutscene background music |
+| [HowToPlayLN - crescent moon island but it's bossa nova](https://soundcloud.com/indekkusu/crescent-moon-island-but-its-bossa-nova) |  | Cutscene background music |
+| [HowToPlayLN - polychromatic dissociation](https://soundcloud.com/indekkusu/polychromatic-dissociation) |  | Cutscene background music |
+| [Imaginary Reality - Stardust Antagonism](https://soundcloud.com/imaginaryrea1ity/stardust-antagonism) |  | Cutscene background music |
+| [Imaginary Reality vs. HowToPlayLN - Throughout The Memories](https://soundcloud.com/indekkusu/throughout-the-memories) |  | Cutscene background music |
+| [takehirotei vs. HowToPlayLN - Essence](https://soundcloud.com/indekkusu/essence) | [#1](https://osu.ppy.sh/beatmapsets/1943786#mania/4021083) | Desempate de la gran final |
+| [takehirotei vs. HowToPlayLN - Essence \~among the constellations\~](https://soundcloud.com/indekkusu/essence-among-the-constellations) |  | Créditos |
```
Comment (juankristal) [inline wiki/osu!_originals/es.md:239]: Same here, should be "Canción de los créditos" or something.

File: wiki/osu!_originals/es.md [hunk 4]
```diff
@@ -223,6 +223,21 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
 | :-: | :-: | :-- |
 | [BilliumMoto x Akiri - Cryochemistry](https://soundcloud.com/billiummoto/cryochemistry)[^fa] | [#1](https://osu.ppy.sh/beatmapsets/1698378) | Desempate de la gran final |
 
+#### ![](/wiki/shared/mode/mania.png) [4 Digit osu!mania World Cup 2023](/wiki/Tournaments/4DM/2023)
+
+| Canción | Beatmap | Notas |
+| :-: | :-: | :-- |
+| [Akiri vs. HowToPlayLN - Cognitohazadrous Ocular Vision Impairing Disease](https://soundcloud.com/indekkusu/akiri-vs-howtoplayln-cognitohazadrous-ocular-vision-impairing-disease) |  | Cutscene background music |
+| [HowToPlayLN - Lost in Space](https://soundcloud.com/indekkusu/4dm2023-trailer) |  | Trailer music |
```
Comment (juankristal) [inline wiki/osu!_originals/es.md:231]: "Música del trailer" or "Tema del trailer" I guess

File: wiki/osu!_originals/fr.md [hunk 5]
```diff
@@ -226,6 +226,21 @@ Ces chansons font partie du [osu! James Landino Collection EP](https://fanlink.t
 | :-: | :-: | :-- |
 | [BilliumMoto x Akiri - Cryochemistry](https://soundcloud.com/billiummoto/cryochemistry)[^fa] | [#1](https://osu.ppy.sh/beatmapsets/1698378) | Tiebreaker des Grandes Finales |
 
+#### ![](/wiki/shared/mode/mania.png) [4 Digit osu!mania World Cup 2023](/wiki/Tournaments/4DM/2023)
+
+| Musique | Beatmap | Notes |
+| :-: | :-: | :-- |
+| [Akiri vs. HowToPlayLN - Cognitohazadrous Ocular Vision Impairing Disease](https://soundcloud.com/indekkusu/akiri-vs-howtoplayln-cognitohazadrous-ocular-vision-impairing-disease) |  | Cutscene background music |
+| [HowToPlayLN - Lost in Space](https://soundcloud.com/indekkusu/4dm2023-trailer) |  | Trailer music |
```
Comment (Pisapou) [inline wiki/osu!_originals/fr.md:234]:
```suggestion
| [HowToPlayLN - Lost in Space](https://soundcloud.com/indekkusu/4dm2023-trailer) |  | Musique du trailer |
```

File: wiki/osu!_originals/fr.md [hunk 6]
```diff
@@ -226,6 +226,21 @@ Ces chansons font partie du [osu! James Landino Collection EP](https://fanlink.t
...
+| [HowToPlayLN - monochroma.](https://soundcloud.com/indekkusu/monochroma) |  | Cutscene background music |
```
Comment (Pisapou) [inline wiki/osu!_originals/fr.md:235]:
```suggestion
| [HowToPlayLN - monochroma.](https://soundcloud.com/indekkusu/monochroma) |  | Musique de fond pendant les cutscenes |
```

File: wiki/osu!_originals/fr.md [hunk 7]
```diff
@@ -226,6 +226,21 @@ Ces chansons font partie du [osu! James Landino Collection EP](https://fanlink.t
...
+| [HowToPlayLN - wandering through the stars](https://soundcloud.com/indekkusu/wandering-through-the-stars) |  | Cutscene background music |
```
Comment (Pisapou) [inline wiki/osu!_originals/fr.md:236]:
```suggestion
| [HowToPlayLN - wandering through the stars](https://soundcloud.com/indekkusu/wandering-through-the-stars) |  | Musique de fond pendant les cutscenes |
```

File: wiki/osu!_originals/fr.md [hunk 8]
```diff
@@ -226,6 +226,21 @@ Ces chansons font partie du [osu! James Landino Collection EP](https://fanlink.t
...
+| [HowToPlayLN - crescent moon island but it's bossa nova](https://soundcloud.com/indekkusu/crescent-moon-island-but-its-bossa-nova) |  | Cutscene background music |
```
Comment (Pisapou) [inline wiki/osu!_originals/fr.md:237]:
```suggestion
| [HowToPlayLN - crescent moon island but it's bossa nova](https://soundcloud.com/indekkusu/crescent-moon-island-but-its-bossa-nova) |  | Musique de fond pendant les cutscenes |
```

File: wiki/osu!_originals/fr.md [hunk 9]
```diff
@@ -226,6 +226,21 @@ Ces chansons font partie du [osu! James Landino Collection EP](https://fanlink.t
...
+| [HowToPlayLN - polychromatic dissociation](https://soundcloud.com/indekkusu/polychromatic-dissociation) |  | Cutscene background music |
```
Comment (Pisapou) [inline wiki/osu!_originals/fr.md:238]:
```suggestion
| [HowToPlayLN - polychromatic dissociation](https://soundcloud.com/indekkusu/polychromatic-dissociation) |  | Musique de fond pendant les cutscenes |
```

File: wiki/osu!_originals/fr.md [hunk 10]
```diff
@@ -226,6 +226,21 @@ Ces chansons font partie du [osu! James Landino Collection EP](https://fanlink.t
...
+| [Imaginary Reality - Stardust Antagonism](https://soundcloud.com/imaginaryrea1ity/stardust-antagonism) |  | Cutscene background music |
```
Comment (Pisapou) [inline wiki/osu!_originals/fr.md:239]:
```suggestion
| [Imaginary Reality - Stardust Antagonism](https://soundcloud.com/imaginaryrea1ity/stardust-antagonism) |  | Musique de fond pendant les cutscenes |
```

File: wiki/osu!_originals/fr.md [hunk 11]
```diff
@@ -226,6 +226,21 @@ Ces chansons font partie du [osu! James Landino Collection EP](https://fanlink.t
...
+| [Imaginary Reality vs. HowToPlayLN - Throughout The Memories](https://soundcloud.com/indekkusu/throughout-the-memories) |  | Cutscene background music |
```
Comment (Pisapou) [inline wiki/osu!_originals/fr.md:240]:
```suggestion
| [Imaginary Reality vs. HowToPlayLN - Throughout The Memories](https://soundcloud.com/indekkusu/throughout-the-memories) |  | Musique de fond pendant les cutscenes |
```

File: wiki/osu!_originals/fr.md [hunk 12]
```diff
@@ -226,6 +226,21 @@ Ces chansons font partie du [osu! James Landino Collection EP](https://fanlink.t
 | :-: | :-: | :-- |
 | [BilliumMoto x Akiri - Cryochemistry](https://soundcloud.com/billiummoto/cryochemistry)[^fa] | [#1](https://osu.ppy.sh/beatmapsets/1698378) | Tiebreaker des Grandes Finales |
 
+#### ![](/wiki/shared/mode/mania.png) [4 Digit osu!mania World Cup 2023](/wiki/Tournaments/4DM/2023)
+
+| Musique | Beatmap | Notes |
+| :-: | :-: | :-- |
+| [Akiri vs. HowToPlayLN - Cognitohazadrous Ocular Vision Impairing Disease](https://soundcloud.com/indekkusu/akiri-vs-howtoplayln-cognitohazadrous-ocular-vision-impairing-disease) |  | Cutscene background music |
```
Comment (Pisapou) [inline wiki/osu!_originals/fr.md:233]:
```suggestion
| [Akiri vs. HowToPlayLN - Cognitohazadrous Ocular Vision Impairing Disease](https://soundcloud.com/indekkusu/akiri-vs-howtoplayln-cognitohazadrous-ocular-vision-impairing-disease) |  | Musique de fond pendant les cutscenes |
```

**Comment by Walavouchey** [comment]
thank you for adding these!

in the future:

- don't pr from the `master` branch, instead [make a new feature branch](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits)
- [apply reviews in batches](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#applying-reviews) from the `Files Changed` tab, like this:

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**